RSSParser/RSSItem.h

```diff
@@ -10,6 +10,12 @@
 
 @interface RSSItem : NSObject <NSCoding>
 
+typedef enum {
```

Comment:
`NS_ENUM` is the recommended means of creating an enum http://nshipster.com/ns_enum-ns_options/